### PR TITLE
LPS-156312 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LPS-156312

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -181,6 +181,58 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "LPS-156312 - Assert an Action can be created with Expression Builder."
+	@priority = "4"
+	test CanCreateActionWithExpressionBuilder {
+		task ("Given an object and field is created") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:187]: Given an object with a field is created

**Review by timpak — APPROVED** [review]
Great job with the test @epark18! I like the usage of Given When Then in the tasks, I'll most likely adopt this for my team. I'm going to make a minor change in the wording in one of the tasks.

**Comment by timpak** [comment]
Added to https://github.com/liferay-bpm-qa/liferay-portal/pull/16.

@epark18 for future PRs, you can send them to liferay-bpm-qa for review. We recently rolled out a new review process [here](https://liferay.atlassian.net/wiki/spaces/DXPL/pages/2134114702/Test+Creation+Guidelines)

**Pull request closed without merging**